docs/Introduction_to_Backend_Architecture.md

```diff
@@ -0,0 +1,123 @@
+# KubeFATE 部署FATE支持后端介绍
```

Comment:
Will there be an English version for this doc?

Reply:
If this is not plan to be done in this pr, then we need to create another issue to track.

Reply:
I will append an English document